src/compiler/commandLineParser.ts

```diff
@@ -7,6 +7,39 @@
 namespace ts {
     /* @internal */
     export const compileOnSaveCommandLineOption: CommandLineOption = { name: "compileOnSave", type: "boolean" };
+
+    /* @internal */
+    export const libMap = createMapFromTemplate({

```

Comment:
This must be a Map to be used in the compiler option I pulled it from, below. If I added a function then we would have yet another place to maintain this list (in addition to the jakefile/gulpfile).